src/configuration/rendererSettings.js

```diff
@@ -86,10 +78,36 @@ export default function rendererSettings() {
                 description: 'Site has responded to issue, DM needs to review.'
             }
         ],
-        filter_cols: ['subset1', 'subset2', 'subset3'],
+        filter_cols: [
+            {
+                value_col: 'subset1',
+                label: 'Subset 1'
+            },
+            {
+                value_col: 'subset2',
+                label: 'Subset 2'
+            },
+            {
+                value_col: 'subset3',
+                label: 'Subset 3'
+            },
+            {
+                value_col: 'subjfreezeflg',
+                label: 'Subject Freeze Status',
+                multiple: true,
+                subject_export: true // I'd'like to add a check that these only change for subject or higher but not sure how i would do so
```

Comment:
good idea.  maybe make a subjectnameoridentifier-subjfreezeflg set and verify all the lengths are 1.